packages/html-reporter/src/reportView.css

```diff
@@ -29,6 +29,24 @@ body {
   width: 100%;
 }
 
+.report-body {
```

Comment:
This probably breaks the mobile view.

Reply:
This is taken directly from `.test-case-column`, which was removed.

<img width="564" alt="Screenshot 2025-04-22 at 8 54 31 AM" src="https://github.com/user-attachments/assets/6a9c40ed-a8fd-4508-a5ef-067fdd55b59b" />